components/create-collective/CreateCollectiveForm.js

```diff
@@ -235,13 +255,15 @@ class CreateCollectiveForm extends React.Component {
                     >
                       {inputProps => (
                         <Field
+                          onClick={() => this.setState({ isURLFieldTouched: true })}
```

Comment:
I'm not sure this will work if you're using the tab key to switch field. Can we rather rely on the `onChange` event?